kustomize/overlays/stable/manager/keycloak-realm.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
YdfnnA/cUaNpzMjeuGaJm+/b/4PakCr+kCarGrqEuxq3gLhtGS0nrPYWmT1bMAImMgEyOR1MKvkAQ5g8wekc/Qa4hpj5pDDnKHPRVHdtrOsPh7vUEPaaWUfzg14vWFBJL6SX38UBCBX/TKKzmP7kqcxY9iTMdXQibHwjEPC8bOOUXzUz+e/2u9HB63MrIm1eTHIEVLuUCBPgx2gCXkP9RHX+FCXScC8BDI3OYTLUlgXbQgPn0N7nlwZ7uNNZKCMq0v/viHLkHz/zBbUv1OU846G3QTBle28x9pik8I6surkQ405fUVI6sSQbJv7kkvn/yTvllVIT68+prsMUBc2t8l9x7affrMBVH2ga3dWZR97M6kCoEyZyZTHnkAXVkh02tslm7rP+O3JgyGKEiKT0aY62Pr3vjb7n589WCyYmzMya2+oapCOtvyneMzzWfFliOvcuYdKvypK5X9jN5ndclcAY5yJvvx/dOrNYPoUcbpgu6+C4JLqxaTDFZpAiBFKC5r0Kghkuv7ekL3T5pDL2m3UoHRXF5YgDhNgDZb70XXBD5QYI1Vwa3bUAMQrA27xqDWplNWd882FMjlMOaZlb094T4SotirRuLReTrSukXkcsiwGBN8/kfhYeDuKvrnH+GqjWp0UuX9PFtDf1PMvYNjkAgyjtYYx+dbCWQLe+NbqV4TVnGBgYXg8uCf2oCjo0XYdS4GPqu8CG/Lh6cpgSefe8qBitA6UluyoIkgkj92M6mgGfy0NNs7Tmu6wTdAZQmMyZZSG+hNZ7L4NyvjQMM8YA5la/4suP9VS08tv8Y2bllsd3TjcnsPUE2O2Rf3awUICrdaRnvAwThtjcZ1RUCgIcQmUsqStTC04qTzrD1GgOz8+PgFOkQXtCOiT+RSVkaWsBnou9FX3KlJIMmJBK4o5ADUojXNpo1+w41hrxyt3cKWXdRCI4l16mfFWHMcCr3WutVUD3cmPc/7Jczy6aHyFdmf2SdYoI2vekzriraWgteu3u8xfI57sSwe75ITWXWFgwLGsZmXDIIarcNnTtdYcuPt6n3NuUa4Glr0bklz4uaDFZN2Kttx/lSiyIs3xzOFuDAt1fn43cj7717mFLa880KaTHva2EcKCNd3jQynntGvjs5BmFB8ZeuPZ/PWq/OMXDE+nglZzBTnZSsYYXXVMNp28fEoSabuQ83nBIYAr7f06w7XXndDPTnXUtCg8dckX8v1LAqOIA7Bxi21WNBTOD4VSXQIbGyj2z9HlBnHf9u4evFGMa43eE34CZnuh4dEmLkvBLflwveYufuAHT6CBI0LvZvCvOKtNeKhuezmRUPBwJVH+lJRjECpx5PZUoF/SHQBvogHPHEWpvakqMxu/XQJQZY5e/WuSELAjxMpEbxILTF3+FSGT2UrIk+cvt7kH4w7+nITVvG888WD0r8w1qU6PL17nOSxAIxLE+NTL6DJC6sju84ffrN0hKdpkQO9RHtHtrxMTGqKT3maCtXvYi/+KviT8MvFoPcm4l8lXayXUSpAhnK3knlXyN0v53R6ym0yuUIIwYfOWl86tlFNgE8wuTAFXGbUXWsRPCHvVsc9OTTU3yspTseiLcbz9TdRkIQ/l4FczPFn2nOz4wKilHMzkDiZk5wlgnZJzSfTBJ0pL9G/XVjY0hSiMk0USGcRL/EEpqRki6mRl5HEzvGQgOnqif0FeGAouSeJF0kUJtJqV2t8iHnzAVu31ZZluaJTADj+QgB/OHBaTBV9thyyhBrT8+DCPrGgwj3Jz0DqHI80KRaMuxbwCMx63iEvH+mmJy1jk9EKVIqeu170wQ29yEOGzdqtW9o0Y8Jsvr+owRCwIr8IZYPJJGkHU62kAjw9pGGbeflQctkvgJ1OT7nBqys5l+SQ3FPITv1z/cMK1PFHLKqFpyaRVNKUt8rDi+FbSbCzz6YZ3/prMMu/oZUs7jC46SsqEGyWR6wZYhLdMP8NkhLB7Hyokv9TpM8FFuAjxu+HMpTAFMy9VubVLJ/QVEcvK6ovshdOPxriMc4rUKzqfPG6UgA5AeaQLzulU/HvYDpyhRmA04YqDZfuzaRyZ0vDc/G2wdKgeXiG8lU9t0vfJYYidVJySQmlB6UqlDMF0ALE4rnSVz3TF5ZfJxz60czW4MpoeZucmG1+FAUqOYvKUvz/SmZcYp/MgMO89/1QEuz1aY+QK/RuyjsrGnFZW/FtaGx1wlWyj0DKeZJBkxcVFFoF1w9HGzurwTLtDWjbO+ZWzqPvrvytU/sef8huH6eyhvMfr2Cj6bp2b6Tudiqxkd7vPoxsjC9WW3tJN9bABhEMAr9Cilo0PO5zJUlSYb+2qtUjEkj1Lb/I5giJylU5leZLFagv1Vi/dda1fFyRcozqClHNyntRbrwvcoKP3dqObL+45eB3i4+m5CQ4wLgApVABUmFJThMxB42Cxbab9npNg84B2a7GDlA7jVR9RjReLE18S8Zsd+YAuPrt8ycVbu/V2Wsw+/wBuIjzsG3au0MhcMZfTIrl/3nMK0fHO4rsK+/HlXq5JkUnPe7J4nvTPy6osDkUerztExBidSCDsn79IqpB+HHxJ7cpV9ze8co6GUf+wUtqsHR02dQ+UHsJD0G4nQceGQs4QQxDWPWwOdQq2A9qm8poWwfbH3TAOtK9mUmfUani2LGIFSFC/1IsYnGmBxQs3fiVI0hGY5kzz98QXhgV6y86PKsh8ss697UTGG4n+NM2pkhDj1bevJ2gBv7PNm92Wadmdx8tO5EDgiWMDA2MIeHt5rrYY0WMtFzLl3t7jKYa/lDJBxoRlK8X8HS4wnGwDTuoDm286hdxZh6tgOycWYKnjkp0Hdu1hUOmuJQfjrRkckQrafFBDiXzx2sKZNXrndavgz00Cp1oA03JYD84uIzkAlJw0yGn+1MWHdx/Wf2LC6BdxbeXdXxmccV4GNBLeqkz7qkIQW2GA/ysaU6zR/7MN/UTbG5sx4qHF097TpEul3Y0dcTWuqJFAx1OyXbw==
```

Comment:
@r00ta Should we do stable in other commit once we're happy this works in `dev`?

Reply:
done